server/src/main/java/io/crate/analyze/AlterTableAddColumnAnalyzer.java

```diff
@@ -123,6 +124,11 @@ public AnalyzedAlterTableAddColumn analyze(AlterTableAddColumn<Expression> alter
                 analyzedTableElements.addCheckColumnConstraint(tableInfo.ident(), check);
                 analyzedTableElementsWithExpressions.addCheckColumnConstraint(tableInfo.ident(), check);
             });
+        GeneratedColumnValidator.validate(
+            addColumnDefinitionWithExpression.generatedExpression(),
+            tableInfo.ident(),
+            analyzedTableElements.columnIdents().iterator().next().name(),
```

Comment:
What if there is more than one column?

Reply:
But we cannot add multiple columns at once with `ALTER TABLE`, can we?
(As far as I see from the grammar we can't, or do I miss something in the syntax?)